### DISCUSSION TITLE
How to show Cluster navigation in EditRecord and CreateRecord pages same as ListRecords page?

### DISCUSSION THREAD

**Discussion by madushancs** [filamentphp/filament#13029] [Help] [answered]
### Package

Panel builder

### Package Version

v3.2.85

### How can we help you?

Hi. Is posible to show cluser navigation in EditRecord and CreateRecord pages as showing in image?

List Page
![image](https://github.com/filamentphp/filament/assets/45415384/300b666e-6a31-40d7-a1bb-d66165562a05)

Edit page
![image](https://github.com/filamentphp/filament/assets/45415384/dc95300d-c0cb-47e2-b023-0c4912d01c81)

Thanks

**Comment by leandrocfe** [comment, marked as answer]
add this in the create page and edit page

```php
public function getSubNavigation(): array
{
    if (filled($cluster = static::getCluster())) {
        return $this->generateNavigationItems($cluster::getClusteredComponents());
    }

    return [];
}
```

**Reply by madushancs** [reply]
Thank you very much. it is working

**Reply by Mohammed-800** [reply]
Thank you very much. it is working bro
